README.md

```diff
@@ -60,3 +60,6 @@ See our contribution guidelines in these languages:
 - [Indonesian](CONTRIBUTING_ID.md)
 
 When contributing to our project, please use English when communicating with other people in issues and/or pull requests. [Click here](CONTRIBUTING.md#why-are-we-using-english-in-our-issues--prs) to read why. ([Bahasa Indonesia](CONTRIBUTING_ID.md#mengapa-kita-menggunakan-bahasa-inggris-dalam-menulis-issue-dan-pull-request))
+
+## API
+for api documentation, [click here](docs/api-reference.md)
```

Comment:
```suggestion
## API

For API documentation, [click here](docs/api-reference.md).
```

As mentioned earlier, please give an extra line between headings and paragraphs.